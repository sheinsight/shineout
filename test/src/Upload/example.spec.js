import exampleTest from '../../example'

exampleTest('Upload', {
  ignore: 'example-03-confirm',
})
