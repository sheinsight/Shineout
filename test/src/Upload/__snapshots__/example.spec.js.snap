// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test: Upload[site/pages/components/Upload/example-01-base.js] 1`] = `
<div>
  <span>
    <button>
      <i />
      <span>
        Upload file
      </span>
    </button>
    <input />
  </span>
</div>
`;

exports[`Snapshot test: Upload[site/pages/components/Upload/example-01-onChange.js] 1`] = `
<div>
  <span>
    <button>
      <i />
      <span>
        Upload file
      </span>
    </button>
    <input />
  </span>
</div>
`;

exports[`Snapshot test: Upload[site/pages/components/Upload/example-02-image.js] 1`] = `
<div>
  <span>
    <span>
      <div>
        <div />
      </div>
      <input />
    </span>
  </span>
</div>
`;

exports[`Snapshot test: Upload[site/pages/components/Upload/example-02-show-image.js] 1`] = `
<div>
  <span>
    <div>
      <div />
    </div>
    <input />
  </span>
</div>
`;

exports[`Snapshot test: Upload[site/pages/components/Upload/example-03-button.js] 1`] = `
<div>
  <span>
    <div>
      <span>
        点击上传
      </span>
    </div>
    <input />
  </span>
</div>
`;

exports[`Snapshot test: Upload[site/pages/components/Upload/example-04-validator.js] 1`] = `
<div>
  <span>
    <div>
      <div>
        <i />
         Upload Image
        <br />
        Allow size 200 * 100
      </div>
    </div>
    <input />
  </span>
</div>
`;

exports[`Snapshot test: Upload[site/pages/components/Upload/example-05-filesize.js] 1`] = `
<div>
  <div>
    <span>
      <button>
        <i />
        <span>
           Upload file
        </span>
      </button>
      <input />
    </span>
  </div>
  <div>
    <span>
      <div>
        <div />
      </div>
      <input />
    </span>
  </div>
</div>
`;

exports[`Snapshot test: Upload[site/pages/components/Upload/example-06-error.js] 1`] = `
<div>
  <div>
    <span>
      <button>
        <i />
        <span>
           Upload file
        </span>
      </button>
      <input />
    </span>
  </div>
  <div>
    <span>
      <div>
        <div />
      </div>
      <input />
    </span>
  </div>
</div>
`;

exports[`Snapshot test: Upload[site/pages/components/Upload/example-07-recover.js] 1`] = `
<div>
  <div>
    <span>
      <button>
        <i />
        <span>
           Upload file
        </span>
      </button>
      <input />
    </span>
  </div>
  <div>
    <span>
      <div>
        <div />
      </div>
      <input />
    </span>
  </div>
</div>
`;

exports[`Snapshot test: Upload[site/pages/components/Upload/example-08-request.js] 1`] = `
<div>
  <span>
    <button>
      <i />
      <span>
         Upload file
      </span>
    </button>
    <input />
  </span>
</div>
`;

exports[`Snapshot test: Upload[site/pages/components/Upload/example-09-zip.js] 1`] = `
<div>
  <span>
    <button>
      <i />
      <span>
         Upload file
      </span>
    </button>
    <input />
  </span>
</div>
`;

exports[`Snapshot test: Upload[site/pages/components/Upload/example-10-defaultValue.js] 1`] = `
<div>
  <div>
    <span>
      <button>
        <i />
        <span>
           Upload file
        </span>
      </button>
      <input />
    </span>
    <div>
      <div>
        test file.png
      </div>
      <span />
    </div>
  </div>
  <div>
    <div>
      <a>
        <div>
          <div>
            <span>
              loading...
            </span>
          </div>
        </div>
      </a>
      <span />
    </div>
  </div>
</div>
`;

exports[`Snapshot test: Upload[site/pages/components/Upload/example-11-dragger.js] 1`] = `
<div>
  <div>
    <span>
      <span>
        <div>
          <div>
            <i />
            <br />
            Click or Drag image to upload
          </div>
        </div>
        <input />
      </span>
    </span>
  </div>
  <br />
  <div>
    <span>
      <span>
        <button>
          <i />
          <span>
              Drop file to upload
          </span>
        </button>
        <input />
      </span>
    </span>
  </div>
</div>
`;
